### ISSUE TITLE
MediaEngineException: Unable set remote SDP for Peer

### ISSUE THREAD

**Issue by bamzoni** [voxeet/android-sdk-toolkit#2]
**Describe the bug**

Voxeet shows overlay but fails to connect to a conference with the following error on some devices (Works fine on Galaxy S8, LG G4, fails on Huawei P20 lite as well as Android Emulator): 

```
E/peerconnection.cc: (line 2332): SetRemoteDescription: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters..
E/Voxeet: Failed to set remote description with error: 
com.voxeet.android.media.MediaEngineException: Unable set remote SDP for Peer: 4652359c-4721-459b-9b46-b0e8d09a9645 :: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters..
W/System.err:     at com.voxeet.android.media.MediaSDK.createAnswerForPeer(MediaSDK.java:74)
 at com.voxeet.sdk.core.abs.ConferenceService$23.onCall(ConferenceService.java:2325)
...

```

**Minimalistic code (recommended)**

```
           val promise = VoxeetToolkit.getInstance().conferenceToolkit.join(intent.getStringExtra(CONFERENCE_KEY))

            if (VoxeetSdk.getInstance().conferenceService.isLive) {
                VoxeetSdk.getInstance().conferenceService
                        .leave()
                        .then(promise)
                        .then(VoxeetSdk.getInstance().conferenceService.startVideo())
                        .error(object : ErrorPromise() {
                            override fun onError(error: Throwable) {
                                Timber.e(error)
                                dismissConference()
                            }
                        })
            } else {
                promise.then(VoxeetSdk.getInstance().conferenceService.startVideo())
                        .error(object : ErrorPromise() {
                            override fun onError(error: Throwable) {
                                Timber.e(error)
                                dismissConference()
                            }
                        })
            }
```

**Specifications**

  - Device: Huawei P20 lite, Android Emulator
  - OS: 9.0, 8.0
  - Platform: Android
  - Version:
    - com.voxeet.sdk:toolkit:2.0.29

**Comment by codlab** [comment]
This device is not a one that we have currently. Normally, the logs should be quite verbose concerning the conference state before it leaves due to this issue.

Can you copy paste every logs before this issue happens? It would help greatly to have a first lead about this device

**Comment by bamzoni** [comment]
@codlab 

```

2019-07-15 16:14:06.408 25597-25708/com.physitrack.physiapp E/media_engine.hpp: (line 195): --- Native add stream
2019-07-15 16:14:06.408 25597-25708/com.physitrack.physiapp E/Voxeet: StreamAdded ! e587c9a7-b03e-44f5-a9bb-95a41544374f
2019-07-15 16:14:06.410 25597-25708/com.physitrack.physiapp D/MediaEngine: Java stream added
2019-07-15 16:14:06.419 25597-25708/com.physitrack.physiapp D/ConferenceService: onStreamAdded: stream for peer e587c9a7-b03e-44f5-a9bb-95a41544374f
2019-07-15 16:14:06.454 25597-25708/com.physitrack.physiapp E/connection_observer.hpp: (line 24): --- SignalingChange
2019-07-15 16:14:06.482 25597-25709/com.physitrack.physiapp E/webrtcvideoengine.cc: (line 620): No video codecs supported.
2019-07-15 16:14:06.486 25597-25708/com.physitrack.physiapp E/peerconnection.cc: (line 5215): Failed to set remote video description send parameters. (INVALID_PARAMETER)
2019-07-15 16:14:06.487 25597-25708/com.physitrack.physiapp E/peerconnection.cc: (line 2371): Failed to set remote offer sdp: Failed to set remote video description send parameters.
2019-07-15 16:14:06.495 25597-25597/com.physitrack.physiapp E/Voxeet: Failed to set remote description with error: 
2019-07-15 16:14:06.496 25597-25597/com.physitrack.physiapp W/System.err: com.voxeet.android.media.MediaEngineException: Unable set remote SDP for Peer: e587c9a7-b03e-44f5-a9bb-95a41544374f :: Failed to set remote offer sdp: Failed to set remote video description send parameters.
2019-07-15 16:14:06.498 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.android.media.MediaSDK.createAnswerForPeer(MediaSDK.java:74)
2019-07-15 16:14:06.498 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.sdk.core.abs.ConferenceService$23.onCall(ConferenceService.java:2325)
2019-07-15 16:14:06.498 25597-25597/com.physitrack.physiapp W/System.err:     at eu.codlab.simplepromise.PromiseInOut$5.run(PromiseInOut.java:202)
2019-07-15 16:14:06.498 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.handleCallback(Handler.java:873)
2019-07-15 16:14:06.498 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
2019-07-15 16:14:06.499 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Looper.loop(Looper.java:193)
2019-07-15 16:14:06.499 25597-25597/com.physitrack.physiapp W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:6669)
2019-07-15 16:14:06.499 25597-25597/com.physitrack.physiapp W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
2019-07-15 16:14:06.499 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
2019-07-15 16:14:06.499 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-07-15 16:14:06.502 25597-25597/com.physitrack.physiapp D/ConferenceService: New mConference user joined with id: e587c9a7-b03e-44f5-a9bb-95a41544374f checking... (ours is e587c9a7-b03e-44f5-a9bb-95a41544374f
2019-07-15 16:14:06.504 25597-25597/com.physitrack.physiapp D/ConferenceService: run: unknown user in stream added
2019-07-15 16:14:06.504 25597-25597/com.physitrack.physiapp W/System.err: com.voxeet.android.media.MediaEngineException: Unable set remote SDP for Peer: e587c9a7-b03e-44f5-a9bb-95a41544374f :: Failed to set remote offer sdp: Failed to set remote video description send parameters.
2019-07-15 16:14:06.504 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.android.media.MediaSDK.createAnswerForPeer(MediaSDK.java:74)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.sdk.core.abs.ConferenceService$23.onCall(ConferenceService.java:2325)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at eu.codlab.simplepromise.PromiseInOut$5.run(PromiseInOut.java:202)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.handleCallback(Handler.java:873)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Looper.loop(Looper.java:193)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:6669)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
2019-07-15 16:14:06.505 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-07-15 16:14:06.540 25597-25597/com.physitrack.physiapp D/io.sentry.android.event.helper.AndroidEventBuilderHelper: Proguard UUIDs file not found.
2019-07-15 16:14:06.738 25597-25597/com.physitrack.physiapp W/System.err: com.voxeet.android.media.MediaEngineException: Unable set remote SDP for Peer: e587c9a7-b03e-44f5-a9bb-95a41544374f :: Failed to set remote offer sdp: Failed to set remote video description send parameters.
2019-07-15 16:14:06.738 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.android.media.MediaSDK.createAnswerForPeer(MediaSDK.java:74)
2019-07-15 16:14:06.738 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.sdk.core.abs.ConferenceService$23.onCall(ConferenceService.java:2325)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at eu.codlab.simplepromise.PromiseInOut$5.run(PromiseInOut.java:202)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.handleCallback(Handler.java:873)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Looper.loop(Looper.java:193)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:6669)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
2019-07-15 16:14:06.739 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-07-15 16:14:06.740 25597-25597/com.physitrack.physiapp D/ConferenceToolkitController: onCall: joining done false
2019-07-15 16:14:06.740 25597-25597/com.physitrack.physiapp D/Promise: then PromiseExec
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/Promise: then PromiseInOut eu.codlab.simplepromise.PromiseInOut@a242aa5
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := eu.codlab.simplepromise.PromiseInOut@7029e7a
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := null
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := null
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := eu.codlab.simplepromise.Promise@4c2d62b
2019-07-15 16:14:06.741 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:06.742 25597-25597/com.physitrack.physiapp D/Promise: resolve eu.codlab.simplepromise.PromiseInOut@7029e7a
2019-07-15 16:14:06.748 25597-25597/com.physitrack.physiapp D/StartVideoPromise: starting video
2019-07-15 16:14:06.760 25597-25597/com.physitrack.physiapp D/com.voxeet.sdk.utils.Validate: hasPermissionInManifest: permission android.permission.CAMERA is in manifest
2019-07-15 16:14:06.762 25597-25597/com.physitrack.physiapp D/ConferenceInformation: isOwnVideoStarted: false
2019-07-15 16:14:06.764 25597-25597/com.physitrack.physiapp D/ConferenceInformation: setOwnVideoStarted: true
2019-07-15 16:14:06.846 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: EglBase14: SDK version: 28. isEGL14Supported: true
2019-07-15 16:14:06.851 25597-25726/com.physitrack.physiapp D/EGL_emulation: eglCreateContext: 0xe6907b20: maj 3 min 0 rcv 3
2019-07-15 16:14:06.862 25597-25726/com.physitrack.physiapp D/EGL_emulation: eglMakeCurrent: 0xe6907b20: ver 3 0 (tinfo 0xcab0dcc0)
2019-07-15 16:14:06.869 25597-25726/com.physitrack.physiapp E/EGL_emulation: eglQueryContext 32c0  EGL_BAD_ATTRIBUTE
2019-07-15 16:14:06.869 25597-25726/com.physitrack.physiapp E/EGL_emulation: tid 25726: eglQueryContext(1863): error 0x3004 (EGL_BAD_ATTRIBUTE)
2019-07-15 16:14:06.872 25597-25597/com.physitrack.physiapp I/org.webrtc.Logging: CameraCapturer: startCapture: 640x480@25
2019-07-15 16:14:06.872 25597-25708/com.physitrack.physiapp E/media_engine.hpp: (line 648): TrackId: bb113849-ffff-0fff-63fd-000000000000
2019-07-15 16:14:06.873 25597-25708/com.physitrack.physiapp E/Voxeet: StreamUpdated !
2019-07-15 16:14:06.873 25597-25708/com.physitrack.physiapp D/MediaEngine: Java stream updated
2019-07-15 16:14:06.874 25597-25708/com.physitrack.physiapp D/ConferenceService: onStreamUpdated: screen for peer e587c9a7-b03e-44f5-a9bb-95a41544374f
2019-07-15 16:14:06.882 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Create new camera2 session on camera 1
2019-07-15 16:14:06.882 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: start
2019-07-15 16:14:06.888 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Available preview sizes: [320x240, 176x144, 1280x720, 640x480, 1280x720]
2019-07-15 16:14:06.888 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Available fps ranges: [[15.0:30.0], [30.0:30.0]]
2019-07-15 16:14:06.889 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Using capture format: 640x480@[15.0:30.0]
2019-07-15 16:14:06.889 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Opening camera 1
2019-07-15 16:14:06.889 25597-25726/com.physitrack.physiapp D/ConferenceService: onCameraOpening: 1
2019-07-15 16:14:06.903 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Camera opened.
2019-07-15 16:14:06.905 25597-25705/com.physitrack.physiapp D/VoxeetHttp: intercept: builder
2019-07-15 16:14:06.905 25597-25705/com.physitrack.physiapp D/TokenResponseProvider: execute: intercept tokenResponse := {access_token := 9jZsbBYiVkiUTPkVpndCAQ==refresh_token := XVOa1y6d0EWanW169F6BPg==}
2019-07-15 16:14:06.905 25597-25597/com.physitrack.physiapp D/ConferenceService: run: unknown user in stream updated
2019-07-15 16:14:06.856 25597-25597/com.physitrack.physiapp W/com.physitrack.physiapp: type=1400 audit(0.0:160): avc: denied { ioctl } for comm=546872656164203078307865363933 path="socket:[301218]" dev="sockfs" ino=301218 ioctlcmd=8927 scontext=u:r:untrusted_app:s0:c89,c256,c512,c768 tcontext=u:r:untrusted_app:s0:c89,c256,c512,c768 tclass=udp_socket permissive=0
2019-07-15 16:14:06.930 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Camera capture session configured.
2019-07-15 16:14:06.932 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Stabilization not available.
2019-07-15 16:14:06.932 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Auto-focus is not available.
2019-07-15 16:14:06.934 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: Camera2Session: Camera device successfully started.
2019-07-15 16:14:06.935 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: CameraCapturer: Create session done. Switch state: IDLE
2019-07-15 16:14:06.937 25597-25726/com.physitrack.physiapp I/org.webrtc.Logging: SurfaceTextureHelper: Setting listener to org.webrtc.-$$Lambda$Camera2Session$CaptureSessionCallback$UDvzHNj8-cAJE1WNByx98pxD9vA@2ca12a0
2019-07-15 16:14:07.000 25597-25597/com.physitrack.physiapp D/SoundPool: onLoadComplete: a new sound is loaded sampleId:=3
2019-07-15 16:14:07.001 25597-25597/com.physitrack.physiapp D/SoundPool: onLoadComplete: loaded ? 3 0
2019-07-15 16:14:07.012 25597-25597/com.physitrack.physiapp W/AudioTrack: AUDIO_OUTPUT_FLAG_FAST denied by server; frameCount 0 -> 179712
2019-07-15 16:14:07.122 25597-25726/com.physitrack.physiapp D/ConferenceService: onFirstFrameAvailable
2019-07-15 16:14:07.223 25597-25597/com.physitrack.physiapp D/ConferenceInformation: setOwnVideoStarted: true
2019-07-15 16:14:07.224 25597-25597/com.physitrack.physiapp D/ConferenceService: onEvent: start video event true null
2019-07-15 16:14:07.228 25597-25597/com.physitrack.physiapp D/Promise: then PromiseExec
2019-07-15 16:14:07.229 25597-25597/com.physitrack.physiapp D/Promise: then PromiseInOut eu.codlab.simplepromise.PromiseInOut@976de59
2019-07-15 16:14:07.232 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:07.233 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := eu.codlab.simplepromise.PromiseInOut@131fa1e
2019-07-15 16:14:07.233 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := null
2019-07-15 16:14:07.233 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:07.233 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:07.233 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := null
2019-07-15 16:14:07.234 25597-25597/com.physitrack.physiapp D/PromiseInOut: having inout parent := eu.codlab.simplepromise.Promise@db0d2ff
2019-07-15 16:14:07.234 25597-25597/com.physitrack.physiapp D/PromiseInOut: executing promise ----------
2019-07-15 16:14:07.236 25597-25597/com.physitrack.physiapp D/Promise: resolve eu.codlab.simplepromise.PromiseInOut@131fa1e
2019-07-15 16:14:07.240 25597-25708/com.physitrack.physiapp E/peerconnection.cc: (line 2332): SetRemoteDescription: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters..
2019-07-15 16:14:07.243 25597-25597/com.physitrack.physiapp E/Voxeet: Failed to set remote description with error: 
2019-07-15 16:14:07.245 25597-25597/com.physitrack.physiapp W/System.err: com.voxeet.android.media.MediaEngineException: Unable set remote SDP for Peer: e587c9a7-b03e-44f5-a9bb-95a41544374f :: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters..
2019-07-15 16:14:07.246 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.android.media.MediaSDK.createAnswerForPeer(MediaSDK.java:74)
2019-07-15 16:14:07.246 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.sdk.core.abs.ConferenceService$7.onCall(ConferenceService.java:928)
2019-07-15 16:14:07.246 25597-25597/com.physitrack.physiapp W/System.err:     at eu.codlab.simplepromise.PromiseInOut$5.run(PromiseInOut.java:202)
2019-07-15 16:14:07.247 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.handleCallback(Handler.java:873)
2019-07-15 16:14:07.247 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
2019-07-15 16:14:07.247 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Looper.loop(Looper.java:193)
2019-07-15 16:14:07.247 25597-25597/com.physitrack.physiapp W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:6669)
2019-07-15 16:14:07.247 25597-25597/com.physitrack.physiapp W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
2019-07-15 16:14:07.247 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
2019-07-15 16:14:07.248 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-07-15 16:14:07.256 25597-25597/com.physitrack.physiapp W/System.err: com.voxeet.android.media.MediaEngineException: Unable set remote SDP for Peer: e587c9a7-b03e-44f5-a9bb-95a41544374f :: Session error code: ERROR_CONTENT. Session error description: Failed to set remote video description send parameters..
2019-07-15 16:14:07.256 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.android.media.MediaSDK.createAnswerForPeer(MediaSDK.java:74)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at com.voxeet.sdk.core.abs.ConferenceService$7.onCall(ConferenceService.java:928)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at eu.codlab.simplepromise.PromiseInOut$5.run(PromiseInOut.java:202)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.handleCallback(Handler.java:873)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:99)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at android.os.Looper.loop(Looper.java:193)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at android.app.ActivityThread.main(ActivityThread.java:6669)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at java.lang.reflect.Method.invoke(Native Method)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
2019-07-15 16:14:07.257 25597-25597/com.physitrack.physiapp W/System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-07-15 16:14:07.272 25597-25597/com.physitrack.physiapp D/AudioSDK: AudioStackManager :: requestAudioFocus in progress com.voxeet.audio.mode.SpeakerMode@852309d
2019-07-15 16:14:07.291 25597-25597/com.physitrack.physiapp D/AudioFocusRequest: requestAudioFocus
2019-07-15 16:14:07.292 25597-25597/com.physitrack.physiapp W/AudioManager: Use of stream types is deprecated for operations other than volume control
2019-07-15 16:14:07.292 25597-25597/com.physitrack.physiapp W/AudioManager: See the documentation of requestAudioFocus() for what to use instead with android.media.AudioAttributes to qualify your playback use case
2019-07-15 16:14:07.296 25597-25597/com.physitrack.physiapp D/AudioSDK: AudioStackManager :: checkOutputRoute in progress
2019-07-15 16:14:07.296 25597-25597/com.physitrack.physiapp D/AudioSDK: BluetoothHeadsetMachine :: checkForBluetoothDeviceAtStartUp
2019-07-15 16:14:07.298 25597-25597/com.physitrack.physiapp E/BluetoothAdapter: Bluetooth binder is null
2019-07-15 16:14:07.298 25597-25597/com.physitrack.physiapp D/AudioSDK: BluetoothHeadsetMachine :: isConnected: false
2019-07-15 16:14:07.301 25597-25597/com.physitrack.physiapp D/AudioSDK: AudioStackManager :: checkOutputRoute: speaker
2019-07-15 16:14:07.311 25597-25597/com.physitrack.physiapp D/AudioFocusRequest: requestAudioFocus
2019-07-15 16:14:07.311 25597-25597/com.physitrack.physiapp W/AudioManager: Use of stream types is deprecated for operations other than volume control
2019-07-15 16:14:07.311 25597-25597/com.physitrack.physiapp W/AudioManager: See the documentation of requestAudioFocus() for what to use instead with android.media.AudioAttributes to qualify your playback use case
2019-07-15 16:14:07.316 25597-25597/com.physitrack.physiapp D/VoxeetConferenceView: run: add viewcom.voxeet.toolkit.implementation.overlays.VoxeetOverlayToggleView{fef8d0a V.E...... ........ 0,220-1080,2380}
2019-07-15 16:14:07.317 25597-25597/com.physitrack.physiapp D/AbstractRootViewProvider: getCurrentActivity: class com.voxeet.toolkit.providers.rootview.DefaultRootViewProvider com.physiapp.communication.IncomingCallActivityNew@eac6bd1
2019-07-15 16:14:07.317 25597-25597/com.physitrack.physiapp D/AbstractRootViewProvider: getCurrentActivity: class com.voxeet.toolkit.providers.rootview.DefaultRootViewProvider com.physiapp.communication.IncomingCallActivityNew@eac6bd1
2019-07-15 16:14:07.317 25597-25597/com.physitrack.physiapp D/DefaultRootViewProvider: getDefaultRootView: com.physiapp.communication.IncomingCallActivityNew@eac6bd1
2019-07-15 16:14:07.320 25597-25597/com.physitrack.physiapp D/AudioSDK: BluetoothHeadsetMachine :: checkForBluetoothDeviceAtStartUp
2019-07-15 16:14:07.321 25597-25597/com.physitrack.physiapp E/BluetoothAdapter: Bluetooth binder is null
2019-07-15 16:14:07.321 25597-25597/com.physitrack.physiapp D/AudioSDK: BluetoothHeadsetMachine :: isConnected: false
2019-07-15 16:14:07.321 25597-25597/com.physitrack.physiapp D/AudioSDK: AudioStackManager :: outputRoute: 

```

I've noticed that sdk tries throws multiple errors and found that it references codec errors. Quick search on stack overflow shows faulty h264 implementation [https://stackoverflow.com/questions/10939321/android-emulator-settings](url). This is a log from emulator, since I don't have the Huawei phone at the moment (although it showed the same exception), I will try to reproduce it tomorrow and paste it here.

**Comment by codlab** [comment]
It's missing a whole lot of lines ^^,

I really meant to have a full list of logs for the app. For security purposes those can be sent to me directly in private if some app is leaking credentials or personnal data

**Comment by codlab** [comment]
A fix is currently being testing, the ETA for a fix is next tuesday/wednesday

The SDK will also have much logs to help investigate for possible future impacted devices. A future release would also had automatic report for us to fix this specific white/blacklist issue

**Comment by codlab** [comment]
a fix is available in the online version 2.0.44

this version also includes ability to add future socks directly without new release is needed and add monitor on our side when such issue should happen in the future for other devices
